packages/go-nogo/examples/index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html>
+
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>jsPsychTimelineGoNogoTimeline Example</title>
+  <script src="https://unpkg.com/jspsych"></script>
+  <script src="../dist/index.global.js"></script>
+  
+  <!-- Load the published timeline package here, e.g.
+<script src="https://unpkg.com/@jspsych-timelines/go-nogo"></script>
+<script src="../dist/index.global.js"></script> -->
+  <link rel="stylesheet" href="https://unpkg.com/jspsych/css/jspsych.css">
+  <!-- <link rel="stylesheet" href="../src/styles.css"> -->
```

Comment:
If we aren't including this, then do we need the styles.css file to be included at all?

Reply:
(As in, should it just be deleted)